src/index.html

```diff
@@ -40,10 +41,56 @@
         </nav>
     </header>
     <div class="tab-content" id="myTabContent">
-        <div class="tab-pane fade show active" id="home" role="tabpanel" aria-labelledby="home-tab">Tab a</div>
+        <div class="tab-pane fade show active" id="home" role="tabpanel" aria-labelledby="home-tab">
+            <button type="button" id="1">Comments</button>
+        </div>
         <div class="tab-pane fade" id="profile" role="tabpanel" aria-labelledby="profile-tab">tab b</div>
         <div class="tab-pane fade" id="contact" role="tabpanel" aria-labelledby="contact-tab">tab c</div>
     </div>
+
+    <!-- Popup -->
+    <div class="popOverlay"></div>
+    <div class="popContainer">
+        <i class="fas fa-times close-pop"></i>
+        <img id='poster' src="#" alt="poster">
+        <h2 class="title">Title</h2>
+        <div class="date-vote">
+            <p class="release_date">Release Date : <span id="date">2021 September 24 </span> </p>
+            <p class="vote_average">Voting Average : <span id="vote">8</span></p>
+        </div>
+        <div class="overview">
+            <h4>Overview : </h4>
+            <p id="overview-txt">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Aut velit amet, corporis ad ab debitis
+                perferendis
+                accusamus doloremque blanditiis culpa voluptatibus fugit iste veritatis reprehenderit sunt esse? Velit,
+                accusantium laboriosam!
+            </p>
+        </div>
+        <div class="comments">
+            <h2 class="comments-title title">Comments (<span class="comments-counter"></span>)</h2>
+            <ul class='comments-list'>
+                <li>
+                    <p>No comments yet 😢 </p>
+                    <p>Add you comment below ⤵️</p>
+                </li>
+            </ul>
+        </div>
+        <div class="add-comment">
+            <h2 class="add-comment title">Add a Comment</h2>
+            <form action="#">
+                <label for="name">Name:</label>
+                <input id="name" type="text" placeholder="Your Name">
+                <label for="comment-description">Comment:</label>
+                <textarea name="comment" id="comment-description" cols="30" rows="6" placeholder="Comment">
+                </textarea>
+                <button id="submit" type="submit">Comment</button>
+            </form>
+        </div>
+    </div>
+
+    <!-- end  Popup -->
```

Comment:
Use Javascript to add this to DOM, It's better our index.html only contains the basic layout which is shared in all parts of the app(header, footer), for sections, popups ... you can use javascript to add it in document load.